src/client/components/movies/Movies.js

```diff
@@ -27,19 +32,24 @@ class Movies extends React.Component {
       )
     }
     if (this.props.movies.length) {
-      const renderMovie = (movie) => { return <Movie key={movie.id} movie={movie} /> }
+      const renderMovie = (movie) => { 
+        const MovieContainer = withIdConnection(Movie);
+        return <MovieContainer key={movie.id} movieDescription={movie} /> 
+      }
       let moviesList = this.props.movies.map(renderMovie);
```

Comment:
you could put `this.props.movies.map(renderMovie)` on the line 45 without creating additional variable moviesList